torch/csrc/jit/tensorexpr/loopnest.h

```diff
@@ -50,6 +50,9 @@ class TORCH_API LoopNest {
   Stmt* getLoopBodyFor(Tensor*) const;
   Stmt* getLoopBodyFor(const Buf*) const;
 
+  //Returns the For stmt indexed by 'indicies' in the 'root' For stmt.
```

Comment:
Can you add some more description of how the `indices` are used to get the `For` stmt? It is not clear unless someone go through the implementation.